### DISCUSSION TITLE
Introducing Möbius Sync Pro for those who want a prepaid version. Also Syncthing core v1.23.1 now on TestFlight

### DISCUSSION THREAD

**Discussion by SimonPickup** [MobiusSync/MobiusSync#97] [Beta]
Two new beta variants of Möbius Sync are now available on TestFlight:

1. The original **Möbius Sync** with in-app purchases. [Sign up for beta here](https://testflight.apple.com/join/n5A7ceuK).
2. A new **Möbius Sync Pro** which will be pre-paid. This was requested by users who have locked-down/enterprise devices that cannot make in-app purchases. [Sign up for beta here](https://testflight.apple.com/join/TzJw0gsD).

Both versions incorporate Syncthing core v1.23.1 and for now, and a few bug fixes: [See details here](https://github.com/MobiusSync/MobiusSync/milestone/6?closed=1).

**Note there is no ability to migrate config/shares/data from one variant of the app to the other. Also note, you cannot run both at once, as they will fight over the network ports.**

Please note this is **beta** software and is not production ready.

**You should only use the beta software in an environment where you can tolerate data corruption or data loss.**

Preferably please send feedback here rather than via TestFlight as we don't get notifications via TestFlight.

**Comment by Taxyovio** [comment]
The app name seems to be too long to be displayed in full on most devices.

**Reply by SimonPickup** [reply]
Do you mean on the home screen? Yes, I might make it just Möbius Sync there, as it's really the same app, just a purchasing mechanism.

**Reply by SimonPickup** [reply]
Fixed, will update TestFlight shortly.

**Comment by rasa** [comment]
@sjpickup Clicking https://testflight.apple.com/join/n5A7ceuK produces
![974F9FF9-7D38-48DA-A11D-0C7533683CE7](https://user-images.githubusercontent.com/220772/218340182-d11f2a68-b2c5-4e28-ada6-9e5b77d2228f.jpeg)

**Reply by SimonPickup** [reply]
Sorry, the group was full from last time we tested a beta - now increased the limit. By the way, the beta for the Freemium version isn't there yet - give Apple another few days to approve. Pro version is available now though, via the other link.

**Reply by SimonPickup** [reply]
Both variants are now available on TestFlight with build 202302121953.

**Comment by SimonPickup** [comment]
Edited this post to add the usual caveat:

Please note this is **beta** software and is not production ready.

**You should only use the beta software in an environment where you can tolerate data corruption or data loss.**

Preferably please send feedback here rather than via TestFlight as we don't get notifications via TestFlight.

**Comment by SimonPickup** [comment]
New TestFlight build 202302140637 available for both Pro and Freemium variants:
- changed App Name on desktop to "Möbius Sync" for both variants
- fixed broken logo in about dialog

**Comment by SimonPickup** [comment]
New TestFlight build 202302190945 is now available for both Pro and Freemium variants:

- Adds initial **EXPERIMENTAL** support for #59. Use at your own risk.

**Comment by SimonPickup** [comment]
New TestFlight build 202302241713 is now available for both Pro and Freemium variants:

  - Hopefully fixes the problem with #59 where permissions would disappear after a day or two. Previous TestFlight build users will have to  reconnect/recreate existing folders. **Still highly experimental - continue to use at your own risk.**
  - Delivers #31 to hep with those long syncs.

**Comment by SimonPickup** [comment]
New TestFlight build 202302270457 is now available for both Pro and Freemium variants:

- I worked out how to get "Open Folder" working appropriately. I have also renamed the button to "Pick External Folder" and added some better help text.
- Set up external app sync as a paid feature, so on the Freemium variant, you will get the in-app purchase dialog when pressing "Pick External Folder" unless already paid.
- There is now an alert if you have both Pro and Freemium versions installed simultaneously.
- The file watcher is now completely disabled on iOS so there will be a more meaningful error message if it is accidentally turned on (e.g. when setting a folder to "Receive Only". (A later step would be to remove the ability to even request it in the GUI.)
- The top-level "Open Folders" button is now "Open Files App". It currently opens at the last folder you visited. I haven't worked out how to force it to go to a top-level folder, e.g. "On My iPhone" yet.

As far as I'm aware, this is now ready to release, so hope to do so within a few days unless any strong negative feedback.

**Comment by SimonPickup** [comment]
Submitted to Apple for review

**Comment by SimonPickup** [comment]
Now approved by Apple and released to the public App Store, both Freemium and Pro versions. Thanks testers!

**Reply by Taxyovio** [reply]
Can you post a link to the prepaid version? App Store is not showing it for some reason.

**Reply by SimonPickup** [reply]
My experience in the past is it can take up to about 24h to propagate around the world. This is supposedly the link, but it's not showing for me (yet) either: https://apps.apple.com/us/app/m%C3%B6bius-sync-pro/id1671184333
